core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/TestCASLockFactory.java

```diff
@@ -375,13 +380,103 @@ public boolean getRemoteRouting()
                             {
                                 return true;
                             }
+
+                            @Override
+                            public String getSerialConsistency(){
```

Comment:
Incorrect formatting {} should be placed on newlines.

Reply:
Done